.github/workflows/test-k8s-lib-injection.yaml

```diff
@@ -8,69 +8,67 @@ on:
     - cron: '0 4 * * *'
 
 jobs:
-  lib-injection-test:
+  injection-tests-nodejs:
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    strategy:
+      matrix:
+        lib-injection-connection: [ 'network','uds']
+        lib-injection-use-admission-controller: ['', 'use-admission-controller']
+        weblog-variant: [ 'sample-app']
+      fail-fast: false
+    env:
+      TEST_LIBRARY: nodejs
+      WEBLOG_VARIANT: ${{ matrix.weblog-variant }}
+      LIBRARY_INJECTION_CONNECTION: ${{ matrix.lib-injection-connection }}
+      LIBRARY_INJECTION_ADMISSION_CONTROLLER: ${{ matrix.lib-injection-use-admission-controller }}
+      DOCKER_REGISTRY_IMAGES_PATH: ghcr.io/datadog/system-tests
+      DOCKER_IMAGE_TAG: ${{ github.sha }}
+      BUILDX_PLATFORMS: linux/amd64
     steps:
-      - name: Checkout repository
+      - name: Checkout nodejs tracer
+        uses: actions/checkout@v3
+        with:
+          path: './dd-trace-js'
+ 
+      - name: Checkout system tests
         uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # 2.3.4
+        with:
+          repository: 'DataDog/system-tests'
+          ref: 'robertomonteromiguel/lib_injection_integration'
```

Comment:
I will remove this:
* When my system-tests branch is merged to main branch
* Just before merging this PR